launchcontainer/launchcontainer.py

```diff
@@ -68,13 +66,53 @@ def student_email(self):
         else:
             return None
 
-    def _get_API_url(self):
-        uri = settings.ENV_TOKENS.get('LAUNCHCONTAINER_API_CONF', DEFAULT_API_CONF)
+    def _validate_conf(self, uri):
+        """Receive a LAUNCHCONTAINER_API_CONF dict and ensure that it is a valid URI.
+
+        This method expects the configuration param to follow this format:
+        """
+
         url_validator = validators.URLValidator()
-        url_validator(uri)
+        try:
+            url_validator(uri)
+        except validators.ValidationError:
+            msg = ("The LAUNCHCONTAINER_API_CONF is not configured properly. "
+                   "Please provide a valid URL.")
+            logger.warning(msg)
+            return None
 
         return uri
 
+    def _get_API_url(self):
+        """Check to see if the URL has been configured in the microsite. If not,
+        fall back to the settings. Finally, default to whatever is defined here."""
+
+        DEFAULT_API_CONF = 'https://wharf.appsembler.com/isc/newdeploy/'
```

Comment:
If we don't have a default, it requires updating server vars to get the XBlock to work. For now, I'd like to leave this, but I will change it during my refactor. (Adding to the card now.)